backend/tests/test_admin_ride_leader.py

```
@@ -0,0 +1,236 @@
+        sample_event.event_date = datetime(2026, 4, 18, 10, 30, tzinfo=timezone.utc)
+        db.commit()
+        _check_in(client, sample_event.id, confirmed_rsvp.id)
+        _mark_leader(client, sample_event.id, confirmed_rsvp.id)
+
+        second_event = RSVP(
+            event_id=sample_event.id,
+            email="duplicate@example.com",
+            name=confirmed_rsvp.name,
+            status="confirmed",
+            privacy_accepted=True,
+            view_token="tok-dup",
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "view_token"

(S106)